apps/src/templates/progress/progressTypes.js

```diff
@@ -43,28 +42,36 @@ const levelWithoutStatusShape = {
   name: PropTypes.string,
   icon: PropTypes.string,
   isUnplugged: PropTypes.bool,
-  isCurrentLevel: PropTypes.bool,
   isConceptLevel: PropTypes.bool,
   pageNumber: PropTypes.number
   /** sublevels: PropTypes.array */ // See below
 };
 // Avoid recursive definition
-levelWithoutStatusShape.sublevels = PropTypes.arrayOf(
-  PropTypes.shape(levelWithoutStatusShape)
+levelSchematicShape.sublevels = PropTypes.arrayOf(
+  PropTypes.shape(levelSchematicShape)
 );
 
-// In the future when the level object does not contain the status object,
-// we can export just levelType without needing levelTypeWithoutStatus.
-export const levelTypeWithoutStatus = PropTypes.shape(levelWithoutStatusShape);
+/**
+ * Going forward, we are moving all user-specific data about a level into
+ * `studentLevelProgressType`, so our `levelType` will just include "schematic"
+ * data that is universal for the level, represented by this type. However, for
+ * now we still need to support the legacy type that includes user-specific
+ * data, which builds on this type.
+ */
+export const levelSchematicType = PropTypes.shape(levelSchematicShape);
```

Comment:
Ultimately, I think we want this type to just be called `levelType` but appreciate that we're taking this intermediate step while we move off of the old levelType.

Reply:
actually scratch that. previously, the "legacy" type got the canonical name (`levelType`) whereas this one had the "special" name, and it didn't occur to me to just flip that. but since `levelSchematicType` is what we ultimately want to be the canonical type, i'll go ahead and rename it now so we don't have to do so later. that also makes the meaning of `levelWithProgressType` a little clearer -- it's a `levelType` + progress.